stores/index.js

```diff
@@ -18,13 +18,13 @@ export default (data) => {
 
   const Reducer = combineReducers({
     apps: AppsStore.makeReducer(data),
```

Comment:
The reason all data is getting passed to the `apps` store is because of here - the reducer just inserts everything from the server into the default state of the `apps` reducer. We should clean that up to only import the stuff it needs.